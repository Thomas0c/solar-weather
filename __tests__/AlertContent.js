import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import AlertContent from '../lib/js/app/components/alertContent';

test('renders correctly', () => {
  const tree = renderer.create(
    <AlertContent
      title="Alert Title"
      description="Alert Description \n Second Line \n"
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
