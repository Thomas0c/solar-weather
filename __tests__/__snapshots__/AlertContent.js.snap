// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityLiveRegion={undefined}
  accessibilityTraits={undefined}
  accessibilityViewIsModal={undefined}
  accessible={undefined}
  collapsable={undefined}
  hitSlop={undefined}
  importantForAccessibility={undefined}
  needsOffscreenAlphaCompositing={undefined}
  onAccessibilityTap={undefined}
  onLayout={undefined}
  onMagicTap={undefined}
  pointerEvents={undefined}
  removeClippedSubviews={undefined}
  renderToHardwareTextureAndroid={undefined}
  shouldRasterizeIOS={undefined}
  style={
    Object {
      "height": "96.00%",
      "padding": 20,
    }
  }
  testID={undefined}
>
  <Text
    accessible={true}
    adjustsFontSizeToFit={undefined}
    allowFontScaling={true}
    ellipsizeMode="tail"
    minimumFontScale={undefined}
    numberOfLines={undefined}
    onLayout={undefined}
    onLongPress={undefined}
    onPress={undefined}
    pressRetentionOffset={undefined}
    selectable={undefined}
    selectionColor={undefined}
    style={
      Object {
        "color": "#343434",
        "fontFamily": "HelveticaNeue",
        "fontSize": 16,
        "fontWeight": "700",
        "marginBottom": 20,
        "textAlign": "center",
      }
    }
    suppressHighlighting={undefined}
    testID={undefined}
    textBreakStrategy={undefined}
  >
    Alert Title
  </Text>
  <RCTScrollView
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityLiveRegion={undefined}
    accessibilityTraits={undefined}
    accessibilityViewIsModal={undefined}
    accessible={undefined}
    alwaysBounceHorizontal={undefined}
    alwaysBounceVertical={undefined}
    automaticallyAdjustContentInsets={undefined}
    bounces={undefined}
    canCancelContentTouches={undefined}
    centerContent={undefined}
    collapsable={undefined}
    contentContainerStyle={
      Object {
        "flexDirection": "column",
      }
    }
    contentInset={undefined}
    contentOffset={undefined}
    decelerationRate={undefined}
    directionalLockEnabled={undefined}
    hitSlop={undefined}
    horizontal={undefined}
    importantForAccessibility={undefined}
    indicatorStyle={undefined}
    keyboardDismissMode={undefined}
    keyboardShouldPersistTaps={undefined}
    maximumZoomScale={undefined}
    mimimumZoomScale={undefined}
    needsOffscreenAlphaCompositing={undefined}
    onAccessibilityTap={undefined}
    onContentSizeChange={undefined}
    onLayout={undefined}
    onMagicTap={undefined}
    onScroll={undefined}
    onScrollAnimationEnd={undefined}
    overScrollMode={undefined}
    pagingEnabled={undefined}
    pointerEvents={undefined}
    refreshControl={undefined}
    removeClippedSubviews={undefined}
    renderToHardwareTextureAndroid={undefined}
    scrollEnabled={undefined}
    scrollEventThrottle={undefined}
    scrollIndicatorInsets={undefined}
    scrollPerfTag={undefined}
    scrollsToTop={undefined}
    shouldRasterizeIOS={undefined}
    showsHorizontalScrollIndicator={undefined}
    showsVerticalScrollIndicator={undefined}
    snapToAlignment={undefined}
    stickyHeaderIndices={undefined}
    style={undefined}
    testID={undefined}
    zoomScale={undefined}
  >
    <View>
      <Text
        accessible={true}
        adjustsFontSizeToFit={undefined}
        allowFontScaling={true}
        ellipsizeMode="tail"
        minimumFontScale={undefined}
        numberOfLines={undefined}
        onLayout={undefined}
        onLongPress={undefined}
        onPress={undefined}
        pressRetentionOffset={undefined}
        selectable={undefined}
        selectionColor={undefined}
        style={
          Object {
            "color": "#343434",
            "fontFamily": "HelveticaNeue",
            "fontSize": 15,
            "textAlign": "left",
          }
        }
        suppressHighlighting={undefined}
        testID={undefined}
        textBreakStrategy={undefined}
      >
        Alert Description 
      </Text>
      <Text
        accessible={true}
        adjustsFontSizeToFit={undefined}
        allowFontScaling={true}
        ellipsizeMode="tail"
        minimumFontScale={undefined}
        numberOfLines={undefined}
        onLayout={undefined}
        onLongPress={undefined}
        onPress={undefined}
        pressRetentionOffset={undefined}
        selectable={undefined}
        selectionColor={undefined}
        style={
          Object {
            "color": "#343434",
            "fontFamily": "HelveticaNeue",
            "fontSize": 15,
            "textAlign": "left",
          }
        }
        suppressHighlighting={undefined}
        testID={undefined}
        textBreakStrategy={undefined}
      >
         Second Line 
      </Text>
      <Text
        accessible={true}
        adjustsFontSizeToFit={undefined}
        allowFontScaling={true}
        ellipsizeMode="tail"
        minimumFontScale={undefined}
        numberOfLines={undefined}
        onLayout={undefined}
        onLongPress={undefined}
        onPress={undefined}
        pressRetentionOffset={undefined}
        selectable={undefined}
        selectionColor={undefined}
        style={
          Object {
            "color": "#343434",
            "fontFamily": "HelveticaNeue",
            "fontSize": 15,
            "textAlign": "left",
          }
        }
        suppressHighlighting={undefined}
        testID={undefined}
        textBreakStrategy={undefined}
      >
        
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
